sodiumpp/include/sodiumpp/sodiumpp.h

```diff
@@ -349,10 +349,13 @@ inline void check_valid_size(T1 current, T2 expected, const char * name, const c
          */
         secret_key() {
             if(P == key_purpose::box) {
-                secret_bytes = locked_string(crypto_box_SECRETKEYBYTES);
+                locked_string tmp(crypto_box_SECRETKEYBYTES); // clang work around
```

Comment:
Seems to also work (at least compile + ./check passes) on debian's clang 3.5.0-10